src/components/issues/Card.tsx

```diff
@@ -107,6 +74,53 @@ const Card: FC<IssueCardProps> = ({ issue }) => {
                     Convert to task
                 </button>
             </div>
+            <p className={styles.card__issue_created__by}>
+                Opened on {created} by
+                <a
+                    href={issue.user.html_url ?? '#'}
+                    target="_blank"
+                    rel="noreferrer"
+                >
+                    {issue.user.login}
+                </a>
+            </p>
+            <div className="card__body">
+                <MarkdownRenderer
+                    className={styles.card__body}
+                    content={issue.body ?? 'No description provided.'}
```

Comment:
did you test for this, when no content is provided? I cant see it in the test file.

Reply:
Done